src/molecules/mixingrules/MixingRuleBase.h

```diff
@@ -3,17 +3,57 @@
 
 #include <string>
 
-class XMLfileUnits;
+#include "utils/Logger.h"
+#include "utils/mardyn_assert.h"
+#include "utils/xmlfileUnits.h"
 
 class MixingRuleBase {
 public:
-	MixingRuleBase(){};
-	virtual ~MixingRuleBase(){};
+	MixingRuleBase(){}
+	virtual ~MixingRuleBase(){}
```

Comment:
I have just deleted it, hope this is ok?

Reply:
no the destructor of a base class should be declared virtual otherwise the whole polymorphic structure does not have cascading destructors. Here, this might not yet be a problem because we don't do any dynamic memory shenanigans but let's be clean and future proof.